### DISCUSSION TITLE
How to use the target argument when using `format-links`?

### DISCUSSION THREAD

**Discussion by aghaynes** [quarto-dev/quarto-cli#10729] [Q&A] [answered]
### Description

In a quarto website, I'd like to provide PDF versions of the various pages, which works great by setting 

```
format:
  html: default
  pdf: default
```

But I would like the PDF to open in a new window (or even appear as a download), which i would normally do with `target="_blank"`. According to the [quarto website](https://quarto.org/docs/output-formats/html-multi-format.html), it's possible to use `format-links` to customise the links, including any of the options from [https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links](the code links section).

Unfortunately, I've been unable to get the target option to work. I've tried all four variants that come to mind (`"blank"`, `"_blank"`, `blank` and `_blank`), but none seem to work for the PDF, nor generic links. I also don't see the target attribute in the generated HTML code...

Am I missing something, or is it ignoring the additional attributes? I don't see the target attribute in the generated HTML code...

The target does work with other-links though via `target: _blank`...

````qmd
---
title: "x"
format: 
  html: default
  typst: default
format-links:
  - html
  - format: typst
    text: PDF
    icon: file-pdf
    target: _blank
  - text: Other Link
    href: https://quarto.org/
    icon: hand-thumbs-down
    target: _blank
other-links:
  - text: Another Link
    href: https://quarto.org/
    icon: hand-thumbs-up
    target: _blank
---

This is a Quarto website.

To learn more about Quarto websites visit <https://quarto.org/docs/websites>.

```{r}
1 + 1
```
````

Thanks for any suggestions!

**Comment by cderv** [comment]
I think `target` attribute is only supported for `other-links` - it is not supported for `format-links`. We don't show example of it either but we say 
> You can also provide format-links items using the same options as [Code Links and Other Links](https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links).

So I can see how this is confusing. We are currently not consistent between the options name and their support.

**Reply by cderv** [reply, marked as answer]
I have created an issue to track. Thank you ! 

- https://github.com/quarto-dev/quarto-cli/issues/10730

**Reply by aghaynes** [reply]
Thanks @cderv!
